cmd/skopeo/utils.go

```diff
@@ -44,12 +47,33 @@ func sharedImageFlags() ([]cli.Flag, *sharedImageOptions) {
 	return []cli.Flag{
 		cli.StringFlag{
 			Name:        "authfile",
-			Usage:       "path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
+			Value:       defaultAuthFile(),
+			Usage:       "path of the authentication file.",
 			Destination: &opts.authFilePath,
 		},
 	}, &opts
 }
 
+func defaultAuthFile() string {
+	authFile := os.Getenv("REGISTRY_AUTH_FILE")
+	if authFile == "" {
+		runtimeDir := os.Getenv("XDG_RUNTIME_DIR")
```

Comment:
No; if the file should be searched for in $XDG_RUNTIME_DIR, that should be a c/image default.

… in fact, it already _is_ the default in c/image, so why is it being added here?

Reply:
This helps the user document the behaviour.  We document this in the current code in skopeo that $XDG_RUNTIME_DIR/containers/conf, I am just expanding the path to match what the usre had.

Reply:
The problem with burying the default in containers image is that it can not be revealed to the user. If you want to add the defaultAuthFile interface to containers image then I would be fine with it.

Reply:
If you want to show the path c/image would use, then call _exactly_ the code in c/image that determines that path, don’t make a copy, please.

(Sure, there might be some conversation about making that path public through an API, although I don’t expect that to be a major concern anyway — but just copy&pasting the logic with no process to keep it the same is long-term worse than not showing the path.)